### ISSUE TITLE
Broken symbolic links in 1.4.0

### ISSUE THREAD

**Issue by algent-al** [bikass/kora#104]
While building latest version for Solus i get this errors. It means that these are broken symlinks
```
[Errno 2] No such file or directory: '/home/build/YPKG/root/korla-icon-theme/install/usr/share/icons/korla/apps/scalable/ubuntu-store.svg'
[Errno 2] No such file or directory: '/home/build/YPKG/root/korla-icon-theme/install/usr/share/icons/korla/apps/scalable/ubuntustore.svg'
[Errno 2] No such file or directory: '/home/build/YPKG/root/korla-icon-theme/install/usr/share/icons/korla/apps/scalable/xfce4-sample-plugin.svg'
```

To verify this you can simply run: `find /path/to/search -xtype l`
```
./kora/apps/scalable/ubuntustore.svg
./kora/apps/scalable/xfce4-sample-plugin.svg
./kora/apps/scalable/ubuntu-store.svg
```

**Comment by bikass** [comment]
I fixed symlinks. 
But that command did not worked  'unknown argument..'

**Comment by algent-al** [comment]
No the command works fine
![broken-symlinks](https://user-images.githubusercontent.com/46356616/104510252-303e5900-55eb-11eb-9d2b-32a89e7d26eb.png)

**Comment by algent-al** [comment]
Fixed by 4e5ec07
